yandextank/plugins/Android/plugin.py

```diff
@@ -4,7 +4,14 @@
 import urllib
 import sys
 import glob
+from multiprocessing import Process
 
+try:
+    import volta
+except Exception:
+    print "Please install volta. https://github.com/yandex-load/volta"
+
+from volta.analysis import grab, uploader
```

Comment:
Maybe this one should be inside try/except then you wouldn't need to ignore F401 for unused import